packages/code-studio/src/main/AppMainContainer.tsx

```diff
@@ -950,7 +885,7 @@ export class AppMainContainer extends Component<
               icon={
                 <span className="fa-layers">
                   <FontAwesomeIcon icon={vsGear} transform="grow-3" />
-                  {isDisconnected && !isAuthFailed && (
+                  {connection == null && (
                     <>
                       <FontAwesomeIcon
                         icon={dhSquareFilled}
```

Comment:
This icon is actually never used anymore. When the connection is null, the loader is shown. This was previously shown when the client was attempting to reconnect.

@dsmmcken Is it fine to not show this little disconnect tag on the settings menu when disconnected? We are showing the "Attempting to reconnect" modal when disconnected, so I think it's pretty clear what the state is

The logic for reconnect is now handled outside of this component, so we would need to create a context or pass down the prop just to show this little badge on the settings icon when there's a giant modal that tells you it's reconnecting.